app/views/posts/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>this is your detail page</h1>
-
+<a href="<%= user_posts_path(@post.author) %>"> 
+  <button id="back-btn">⬅</button>
+</a>
+<h3>Post <%= "#" + @post.id.to_s %> by <%= @post.author.name %></h3>
+<div class='post-interactions'>
+  <span>Comments: <%= @post.comments_counter %></span>
+  <span>Likes: <%= @post.likes_counter %></span>
+</div>
+<p><%= @post.content %></p>
+<h3>Comments:</h3>
+<% @post.comments.each do |comment| %>
+  <div class="post-comments">
+    <span><%= comment.author.name %>:</span>
+    <span><%= comment.text %></span>
+  </div>
+<% end %>
```

Comment:
| Your Design | Microverse Design |
|-|-|
| ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/0e0a4c31-98e4-4b33-8874-e5c1ee40a454) | ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/31b54ea8-c7dd-402b-a964-ec16252a2212) |


- [ ] It would be nice if you double-check the original design given by microverse and follow that design while making your views to make your UI professional and attractive 👍
- [ ] Kindly follow the below details 👍

1. Kindly fix this problem of the content because i don't see any content of the post in this view 🤔
2. Kindly focus on the comments_counter and likes_counter because they don't show any number 😨
3. It would be great if you add the commenter name and the comment which a commenter left in this view to follow the given wireframe 😊